client/src/components/commonComponents/Input/index.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import './index.css';
+
+const Input = ({ type, name, label, placeholder, value, onChange }) => {
+  return (
+    <input
+      type={type}
+      name={name}
+      aria-label={label}
```

Comment:
remove this and try to wrap the input with Lable

Reply:
I wrapped the input within a div